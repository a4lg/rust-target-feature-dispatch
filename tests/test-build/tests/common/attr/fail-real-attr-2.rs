pub const VALUE: i32 = target_feature_dispatch::target_feature_dispatch! {
    // Regular cfg attribute but this position only allows pseudo-attributes
    // parsed by the macro.
    #[cfg_attr(all(), allow(unexpected_cfgs))]
    if (()) {
        (1)
    } else {
        0
    }
};
fn main() {}
