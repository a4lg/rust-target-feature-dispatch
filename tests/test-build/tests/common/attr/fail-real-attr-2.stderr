error: invalid pseudo-attribute: #[cfg_attr(all(), allow(unexpected_cfgs))]
  --> tests/common/attr/fail-real-attr-2.rs:1:24
   |
1  |   pub const VALUE: i32 = target_feature_dispatch::target_feature_dispatch! {
   |  ________________________^
2  | |     // Regular cfg attribute but this position only allows pseudo-attributes
3  | |     // parsed by the macro.
4  | |     #[cfg_attr(all(), allow(unexpected_cfgs))]
...  |
10 | | };
   | |_^
   |
   = note: this error originates in the macro `$crate::target_feature_dispatch` which comes from the expansion of the macro `target_feature_dispatch::target_feature_dispatch` (in Nightly builds, run with -Z macro-backtrace for more info)
